backend/open_webui/utils/middleware.py

```
@@ -486,6 +413,100 @@ async def chat_web_search_handler(
+    await __event_emitter__(
+        {
+            "type": "status",
+            "data": {"description": "Generating an image", "done": False},
+        }
+    )
+
+    messages = form_data["messages"]
+    user_message = get_last_user_message(messages)
+
+    prompt = user_message
+    negative_prompt = ""
```

Ruff (0.8.2)

Local variable `negative_prompt` is assigned to but never used

Remove assignment to unused variable `negative_prompt`

(F841)